scripts/docker/web/Dockerfile

```diff
@@ -0,0 +1,129 @@
+    cd /opt/thrift-0.12.0; \
+    ./configure --prefix=/opt/thrift --silent; \
+    cd /; \
+    make -C /opt/thrift-0.12.0 -j4 --silent install; \
+    rm -rf /opt/thrift-0.12.0 /opt/thrift-0.12.0.tar.gz; \
+    # Build GTest.
+    cd /usr/src/googletest/ && \
+    mkdir build && \
+    cd build && \
+    cmake .. && \
+    make install && \
+    cd / && \
+    rm -rf /usr/src/googletest/build
+
+ENV PATH="/opt/thrift/bin:$PATH"
+ENV CMAKE_PREFIX_PATH="/opt/thrift:$CMAKE_PREFIX_PATH"
+
+# Download CodeCompass release.
+RUN git clone https://github.com/Ericsson/CodeCompass.git /CodeCompass
+WORKDIR /CodeCompass
+RUN git checkout ${CC_VERSION}
+
+# Build CodeCompass.
+RUN mkdir /CodeCompass-build && \
+  cd /CodeCompass-build && \
+  cmake /CodeCompass \
+    -DDATABASE=$CC_DATABASE \
+    -DCMAKE_INSTALL_PREFIX=/CodeCompass-install \
+    -DCMAKE_BUILD_TYPE=$CC_BUILD_TYPE && \
+  make && \
+  make install
+
+###############################################################################
+#--------------------------    PRODUCTION STAGE   ----------------------------#
+###############################################################################
+FROM ubuntu:bionic
+
+ARG CC_GID=960
+ARG CC_UID=960
+
+ENV CC_GID ${CC_GID}
+ENV CC_UID ${CC_UID}
+
+ENV TINI_VERSION v0.18.0
+
+RUN set -x && apt-get update -qq \
+  && apt-get install -qqy --no-install-recommends \
+    llvm-7 \
+    libboost-filesystem-dev libboost-log-dev libboost-program-options-dev \
+    odb libodb-sqlite-dev libodb-pgsql-dev \
+    libsqlite3-dev \
+    default-jre \
+    libgit2-dev \
+    libssl1.0.0 \
+    libgvc6 \
+    # To switch user and exec command.
+    gosu \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/ \
+  && set +x
+
+# Copy CodeCompass installed directory.
+COPY --from=builder /CodeCompass-install /codecompass
+
+# Copy thrift installed directory.
+COPY --from=builder /opt/thrift /opt/thrift
+
+# Create user and group for CodeCompass.
+RUN groupadd -r codecompass -g ${CC_GID} \
+  && useradd -r --no-log-init -M -u ${CC_UID} -g codecompass codecompass
+
+# Change permission of the CodeCompass package.
+RUN chown codecompass:codecompass /codecompass
+
+ENV PATH="/codecompass/bin:$PATH"
+ENV LD_LIBRARY_PATH="/codecompass/lib:/opt/thrift/lib:$LD_LIBRARY_PATH"
+
+COPY ./entrypoint.sh /usr/local/bin/
+RUN chmod a+x /usr/local/bin/entrypoint.sh \
+  && chown codecompass:codecompass /usr/local/bin/entrypoint.sh
+
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+
+EXPOSE 8080
+
+ENTRYPOINT ["/tini", "--", "/usr/local/bin/entrypoint.sh"]
+
+# Unfortunately there is an error when using CodeCompass_webserver binary
+# from PATH. See: https://github.com/Ericsson/CodeCompass/issues/353.
+# If this issue will be fixed we can use only the binary name instead of
+# full path.
+CMD ["/codecompass/bin/CodeCompass_webserver", "-w", "/workspace", "-p", "8080", "-d", "sqlite:"]
```

Comment:
This default command will raise a runtime error due to the incomplete connection string.

Reply:
It will not raise runtime error because we do not use this connection string and it will be removed in #359.